WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -424,6 +452,18 @@ - (void)showViewAdminForBlog:(Blog *)blog
     [[UIApplication sharedApplication] openURL:[NSURL URLWithString:dashboardUrl]];
 }
 
+- (void)showRemoveSiteForBlog:(Blog *)blog
+{
+    NSString *model = [[UIDevice currentDevice] localizedModel];
+    NSString *title = [NSString stringWithFormat:NSLocalizedString(@"Are you sure you want to continue?\n All site data will be removed from your %@.", @"Title for the remove site confirmation alert"), model];
```

Comment:
Should we include a note in the localized string description saying what the `%@` in the string is being used for?